insights/v1alpha1/0000_10_01_datagather.crd.yaml

```diff
@@ -0,0 +1,251 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    api-approved.openshift.io: https://github.com/openshift/api/pull/1365
+    include.release.openshift.io/ibm-cloud-managed: "true"
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
+    release.openshift.io/feature-set: TechPreviewNoUpgrade
+  name: datagathers.insights.openshift.io
+spec:
```

Comment:
Currently you have no additional printer columns, what additional columns do you think would be helpful to print out?

Reply:
Added. Thanks.